packages/py3dns/PKGBUILD

```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-dns
```

Comment:
```diff
-pkgname=python-dns
+pkgname=python-py3dns
```

Reply:
dose this applies on any python library with name like "pyfoo" ?

Reply:
Yes.
You have example in AL, if a Pypi package is named `python-multipart` then the packake name should be `python-python-multipart`.
https://archlinux.org/packages/community/any/python-python-multipart/
https://archlinux.org/packages/community/x86_64/python-python-pkcs11/